dev/tasks/python-wheels/github.linux.yml

```diff
@@ -141,9 +139,9 @@ jobs:
       {{ macros.github_upload_wheel_scientific_python("arrow/python/repaired_wheels/*.whl")|indent }}
 
       {% if arrow.is_default_branch() %}
```

Comment:
We could temporarily remove this check for the `default_branch` on the PR to validate it pushes the images correctly

Reply:
Wouldn't it need credentials to push?

Reply:
now I am curious but I am pretty sure those credentials are the ones from the crossbow repo? that's an example of why only collaborators can request archery jobs

Reply:
Ah, let's try it then!

Reply:
Ok, trying now.

Reply:
Well, it went fine!

Reply:
cool, happy to merge once the temporary disabling is reverted!